JetUtilities/src/ClosureMaker.cc

```diff
@@ -533,10 +534,10 @@ void ClosureMaker::makeCanvases() {
 }
 
 //______________________________________________________________________________
-void ClosureMaker::makeMergedCanvas() {
+void ClosureMaker::makeMergedCanvas(bool finemerge = false) {
```

Comment:
I don't love this finemerge variable. To me it just seems like a way to force the canvas range to be narrower. Can't you just change the settings in the function "determineCanvasRange"? Make that function smarter rather than coming up with a way to circumvent it.

Reply:
Why not make it so that if the dynamic range of the plot is too large it duplicates the plot and makes a "full range" version and a "zoomed in" version? Might be hard, but would make more sense from an ease of use and automation standpoint.

Reply:
It is hard to define 'too large'. Actually I think it would be best that we always have such 1.05-0.95 range canvas. So the too large would be defined as 'xmin<0.95 || xmax > 1.05'. And these days, it is almost always the case.